nwbinspector/checks/time_series.py

```diff
@@ -68,6 +68,15 @@ def check_timestamps_ascending(time_series: TimeSeries, nelems=200):
         return InspectorMessage(f"{time_series.name} timestamps are not ascending.")
 
 
+@register_check(importance=Importance.BEST_PRACTICE_VIOLATION, neurodata_type=TimeSeries)
+def check_resolution(time_series: TimeSeries):
+    """Check the resolution value of a TimeSeries for proper format (-1.0 or NaN for unknown)."""
+    if not (np.isnan(time_series.resolution) or time_series.resolution == -1.0) and time_series.resolution <= 0:
```

Comment:
```suggestion
    if np.isnan(time_series.resolution) or time_series.resolution == -1.0:
        return
    if time_series.resolution <= 0:
```

@bendichter Do you think this is less wonky?